bots_utils.py

```diff
@@ -39,3 +40,98 @@ def verify_scope_is_cidr(rule):
     ip = re.split('/|\.', str(rule[SCOPE]))  # break ip to blocks
     rule[SCOPE] = ip[0] + '.' + ip[1] + '.' + ip[2] + '.' + ip[3] + '/' + ip[4]
     pass
+
+
+"""
+Check if two scopes intersect , if it does returns true 
+"""
+
+
+def is_two_scopes_overlap_ipv4(scope1, scope2):
+    n1 = ipaddress.IPv4Network(scope1)
+    n2 = ipaddress.IPv4Network(scope2)
+    intersect = n2.overlaps(n1)
+    if intersect:
+        return True
+    else:
+        return False # cidr if out of scope bounds
+
+
+"""
+Check if cider is completely inside scope(other cidr), if it does returns true  
+"""
+
+
+def is_scope_contained_by_other_ipv4(scope, other):
+    n1 = ipaddress.IPv4Network(scope)
+    n2 = ipaddress.IPv4Network(other)
+
+    inside = n2.supernet_of(n1)
+    if inside:
+        return True
+    else:
+        return False # cidr if out of scope bounds
+
+
+"""
+Check if cider is completely inside scope, if it does returns true 
+"""
+
+
+def is_scope_contained_by_other_ipv6(scope, other):
+    n1 = ipaddress.IPv6Network(scope)
+    n2 = ipaddress.IPv6Network(other)
+    inside = n2.supernet_of(n1)
+    if inside:
```

Comment:
You can use ternary operator:
return True if inside else False

Reply:
done

Reply:
I don't see changes, did you push?